sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -328,10 +328,11 @@ class command_graph {};
 template<>
 class command_graph<graph_state::modifiable> {
 public:
-  command_graph(const device& syclDevice, const property_list& propList = {});
+  command_graph(const context& syclContext, const device& syclDevice,
+  const property_list& propList = {});
```

Comment:
Slight preference for indenting this parameter
```suggestion
  command_graph(const context& syclContext, const device& syclDevice,
                const property_list& propList = {});
```

Reply:
Done!